pkgs/applications/misc/logseq/default.nix

```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
   dontBuild = true;
 
   nativeBuildInputs = [ makeWrapper autoPatchelfHook ];
-  buildInputs = [ stdenv.cc.cc curl expat openssl zlib ];
+  buildInputs = [ stdenv.cc.cc curl expat git openssl zlib ];
```

Comment:
I think git should go in the postFixup phase bellow:
```diff
--- i/pkgs/applications/misc/logseq/default.nix
+++ w/pkgs/applications/misc/logseq/default.nix
@@ -38,6 +38,7 @@ stdenv.mkDerivation rec {
 
   postFixup = ''
     makeWrapper ${electron}/bin/electron $out/bin/${pname} \
+      --prefix PATH : ${lib.makeBinPath [ git ]} \
       --add-flags $out/share/${pname}/resources/app
   '';
 ```
besides that, LGTM

Reply:
Thanks! Added!

Reply:
I forgot to mention that you also need to remove `git` from the buildInputs

Reply:
ah, updated!